libnavigation-core/src/main/java/com/mapbox/navigation/core/directions/session/MapboxDirectionsSession.kt

```diff
@@ -65,6 +65,8 @@ class MapboxDirectionsSession(
             override fun onResponse(routes: List<DirectionsRoute>) {
                 currentRoutes = routes
                 state = State.RoutesAvailable
+                // start the timer only when the route has been requested at least once
+                fasterRouteTimer.start()
```

Comment:
NIT: how about we move this to https://github.com/mapbox/mapbox-navigation-android/pull/2408/files#diff-009d81d6b9a5e6e4e15aa02120bf4aa1L35?